gulpfile.js

```diff
@@ -36,6 +29,8 @@ try {
 
 var TESTAPIURL = "//api.lytics.io";
 var TESTCSSURL = "//c.lytics.io/static/pathfora.min.css";
+var EXAMPLESSRC = "docs/docs/examples/src",
```

Comment:
might as well list all or none of these in comma notation (just for consistency)